packit_service/config.py

```diff
@@ -146,25 +144,28 @@ def get_service_config(cls) -> "ServiceConfig":
         return cls.service_config
 
 
-class PackageConfigGetterForGithub:
+class GeneralPackageConfigGetter:
```

Comment:
We should also have SergeantPackageConfigGetter

Reply:
aye aye captain ;-)

Reply:
i suggest BasePackgeConfigGetter, "Base" is used at least in ogr for similar, so to keep naming convention

Reply:
If it's a naming discussion, I must join!

Would calling it only `PackageConfigGetter` be good enough?

Reply:
Oh no. Naming...;(

My notes regardning the suggestion. (But I can pick any of these if you want.)

> `BasePac...`
- In OGR we use it between abstract class and the real implementation to share the code. Here, we are using this class as well.

> `PackageCo...` 
- This makes sense to me when we don't have any other implementation. (Like we have for Pagure now.)

Reply:
OK, as I am thinking about that. I'll merge those two classes to `PackageConfigGetter` and solve the difference in another way.